presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -89,6 +89,11 @@ Operations
 
     Returns the closure of the combinatorial boundary of this geometry.
 
+.. function:: ST_Buffer(Geometry, double) -> Geometry
```

Comment:
This seems unrelated to the introduction of bing tiles functions. Move it to a separate commit.